tests/component/test_trigger_properties.py

```diff
@@ -0,0 +1,118 @@
+"""Contains a collection of pytest tests that validates trigger properties."""
+
+import pytest
+
+import nidaqmx
+from nidaqmx.constants import TriggerType
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.task import Task
+
+
+@pytest.fixture(scope="module")
+def ai_voltage_task(any_x_series_device):
+    """Gets AI voltage task."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        yield task
+
+
+def test__trigger__get_float_property__returns_default_value(ai_voltage_task: Task):
```

Comment:
I don't think "given trigger" makes sense, because the test is not given a trigger. Rename all test cases to `test__ai_task__...`

Reply:
Renamed all the test cases to `test__ai_task__...`